src/django/api/reports/percent_facilities_with_multiple_matches.sql

```diff
@@ -0,0 +1,30 @@
+SELECT
+    zmonth,
+    (COUNT(*)*100)/total as percent_of_data
+FROM (
+    SELECT i.id, COUNT(fm.id) as match_count, zmonth
+    FROM api_facilitylistitem i
+    JOIN (
+        SELECT
+            fm.id, fm.facility_list_item_id,
+            to_char(fm.created_at, 'YYYY-MM') as created_month,
+            z.month as zmonth
+        FROM api_facilitymatch fm
+        JOIN (
+            SELECT to_char(m.created_at, 'YYYY-MM') as month
+            FROM api_facilitymatch m group by month
+        ) z ON to_char(fm.created_at, 'YYYY-MM') <= z.month
+    ) as fm ON i.id = fm.facility_list_item_id
+    GROUP BY zmonth, i.id
+) as c
+JOIN (
+    SELECT month, count(*) as total
+    FROM api_facilitylistitem i
+    JOIN (
+        SELECT to_char(m.created_at, 'YYYY-MM') as month
+        FROM api_facilitymatch m group by month
+    ) z ON to_char(i.created_at, 'YYYY-MM') <= z.month
+    GROUP BY month
+) t ON t.month = c.zmonth
+WHERE  c.match_count > 1
+GROUP BY zmonth, total;
```

Comment:
I'd like to pair with you and talk through this one. The output looks reasonable, but I am having a little trouble following the logic.

Reply:
I've gotten our changes to this pushed in e6e22bc (the final error we were seeing was just a misplaced comma). The development data result looks significantly better now, and I feel much more convinced it's accurate.

Reply:
And belatedly, a few changes to names for this to clarify the data representations for the users - see b3759f0